### PR TITLE
feat get_or_none method

### PR DESCRIPTION
**Pull request by mishaga**
Closes issues #170 and #151

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
             'aiopg',
             'pytest',
             'pytest-asyncio',
+            'cryptography',
```
Comment (kalombos) [inline setup.py:48]: Why would you need this dependency?

File: tests/test_obsolete.py [hunk 2]
```diff
@@ -297,6 +297,22 @@ async def test(objects):
...
+            obj2 = await objects.get_or_none(TestModel, text=text1)
+            obj3 = await objects.get_or_none(TestModel, text=text2)
+
+            self.assertEqual(obj1, obj2)
+            self.assertIsNotNone(obj1)
+            self.assertIsNotNone(obj2)
+            self.assertIsNone(obj3)
+
+        self.run_with_managers(test)
+
```
Comment (kalombos) [inline tests/test_obsolete.py:315]: Thank you for your PR but could you move this test to test_common.py file and rewrite in a pytest way like other ones? The file contains deprecated tests which have to be rewritten  #177

File: setup.py (hunk 1 above)
Comment (kalombos) [reply setup.py:48]: I've sorted out it is for mysql auth.  resolved

File: tests/test_obsolete.py (hunk 2 above)
Comment (kalombos) [reply tests/test_obsolete.py:315]: I' ve sorted out it's for mysql auth. Resolved

Comment (mishaga) [reply tests/test_obsolete.py]: done

**Pull request merged**